etc/kayobe/seed.yml

```diff
@@ -106,11 +106,14 @@ seed_pulp_container:
     image: pulp/pulp
     pre: "{{ kayobe_config_path }}/containers/pulp/pre.yml"
     post: "{{ kayobe_config_path }}/containers/pulp/post.yml"
-    tag: "{{ '3.21-https' if pulp_enable_tls | bool else '3.21' }}"
+    tag: "{{ '3.22-https' if pulp_enable_tls | bool else '3.22' }}"
```

Comment:
Noting concerns about changes to distribution updates raised in slack https://stackhpc.slack.com/archives/C044YBYMJBZ/p1677077026176839

Reply:
I hit that on 3.21 (not tried 3.22 yet as I didn't want to deviate from the upgrade plan). The issue was that the distributions I had created for the local image build feature deviated from the ones that actually landed in https://github.com/stackhpc/stackhpc-kayobe-config/pull/342. This caused:

```
failed: [localhost] (item={'name': 'treasuredata-4-development', 'base_path': 'treasuredata/4/redhat/8/x86_64/development', 'repository': 'TreasureData 4', 'state': 'present', 'required': True}) => changed=false                         
  ansible_loop_var: item                                                                                                                                                                                             
  item:                                                                                                                                                                                                                                     
    base_path: treasuredata/4/redhat/8/x86_64/development                                                                                                                                                                                   
    name: treasuredata-4-development                                                                                                                                                                                                        
    repository: TreasureData 4                                                                                                                                                                                                              
    required: true                                                                                                                                                                                                                          
    state: present                                                                                                                                                                                                   
  msg: 'HTTP Error 400: Bad Request b''{"base_path":["This field must be unique."]}''' 
```
I guess the other option is that we use the `pull` option on the docker_container module, but that can be non-desirable. We just need a newer build of the container to make sue we have the `PULP_*_WORKERS` variables available (You might have an old version of 3.21 that is missing this feature).

Reply:
Ack. Has 3.22 been tested?

Reply:
Not yet. Is that a requirement for merging?

Reply:
Yes :) ideally on a system with some real data

Reply:
We are testing it now.

Reply:
We've tested 3.22 now and it seems to be working fine :+1:

Reply:
Thanks guys - marking as resolved :)